api/scpca_portal/models/computed_file.py

```diff
@@ -78,6 +78,7 @@ class OutputFileFormats:
     format = models.TextField(choices=OutputFileFormats.CHOICES)
     includes_merged = models.BooleanField(default=False)
     modality = models.TextField(choices=OutputFileModalities.CHOICES)
+    public = models.BooleanField(default=False)
```

Comment:
Let's override the default `objects` manager and replace it with one that the default queryset has `public=True`. 

We can set `all_objects` to be the default manager which would apply to public and private computed files.

Ref: https://docs.djangoproject.com/en/3.2/topics/db/managers/